pkg/log/log.go

```diff
@@ -59,6 +59,11 @@ func Fatalf(format string, args ...interface{}) {
 	logger.log.Fatalf(format, args...)
 }
 
+// Warn logs at warn level
+func Warn(format string, args ...interface{}) {
```

Comment:
this should be `Warnf` ...however we probably want both versions `Warn` and `Warnf`

Reply:
Yes, we want both. I should get rid of the formatting here since the correct use is Warn. I am adding Warnf in a different PR.